src/test/java/uk/ac/ebi/eva/pipeline/jobs/GenotypedVcfJobTest.java

```diff
@@ -146,6 +146,7 @@ public void fullGenotypedVcfJob() throws Exception {
                 .vepCacheVersion("")
                 .vepNumForks("")
                 .vepPath(getResource(MOCK_VEP).getPath())
+                .addString(JobParametersNames.OUTPUT_DIR_STATISTICS, outputDir)
```

Comment:
Please create a utility method to ensure consistency. Applies to other job test classes.